.github/workflows/rust-cubesql.yml

```diff
@@ -107,13 +112,18 @@ jobs:
       image: cubejs/rust-cross:${{ matrix.target }}-02042024
 
     steps:
+      - name: install Curl
+        run: |
+          apt-get update
```

Comment:
It's not a good practice to install (with an update) a software that can be used across all runs in prepared images such as cross. It's our images.

Reply:
Emm.... 
This job runs on: 
```
  native_linux:
    needs: [lint]
    runs-on: ubuntu-20.04
   ....
    Steps:
   ....
      - name: Install Rust
        uses: actions-rust-lang/setup-rust-toolchain@v1

```
And all the setup steps are applied on the host system, not inside docker in the docker.

Reply:
It uses image

```
image: cubejs/rust-cross:${{ matrix.target }}-02042024
```